### ISSUE TITLE
Forever Free GCloud Setup

### ISSUE THREAD

**Issue by FloatingMind12** [antoniomika/sish#238]
When I tried to create a VM instance on Google Cloud Shell, it requested a billing account. There's the free trial but your instances get suspended when the trial period is over. I heard something about `f1-micro` instances being free forever but according this [post](https://stackoverflow.com/questions/58756624/is-f1-micro-vm-machine-type-forever-free), that was already updated a while ago and now only `e2-micro` instances are included in the free tier and only in specific regions.

So my questions are: **Do I have to start a free trial?** since it requests credit card info, I'm a bit reluctant to do it. And **what type of instance should I choose** after all?
I know my issue is a bit unrelated but any answers would be much appreciated

**Comment by antoniomika** [comment]
Based on this: https://cloud.google.com/free/docs/free-cloud-features#compute you are correct, it looks like e2-micro in us-{west,central,east}1 are only covered by the free tier. 

sish isn't really resource intensive so an e2-micro instance should be more than enough! You should be good to go not enabling the trial!

**Comment by FloatingMind12** [comment]
Thanks for the answer but now I have a completely different issue
I always get `Connection closed by xxx.xxx.xxx.xxx port 2222` every time I start the tunnel. Did I misconfigured anything?

**Comment by FloatingMind12** [comment]
I found the answer in #156 but it's still not perfect
I don't know why but even after I set `--cleanup-unbound-timeout` to 2m and even 5m, I still get the `closed network connection` error every now and then
How could it be even though ssh login only lasts for seconds

And sorry but I don't understand your explanation in the other thread according
`Aborting with error cannot find connection for host: xxx.xxx.xxx.xxx`
Does it means that there is just no fix?
